apps/src/lib/kits/maker/BoardController.js

```diff
@@ -16,9 +16,23 @@ import {BOARD_EVENT_ALIASES} from './PlaygroundConstants';
 import TouchSensor from './TouchSensor';
 import Piezo from './Piezo';
 
+/**
+ * @typedef {Object} SerialPortInfo
+ * @property {string} comName (a port id) e.g. "COM3" or "/dev/ttyACM0"
+ * @property {string} manufacturer e.g. "Adafruit Circuit Playground"
+ * @property {string} productId e.g. "0x8011"
+ * @property {string} vendorId e.g. "0x239a"
+ */
+
 /** @const {string} */
 const CHROME_APP_ID = 'ncmmhcpckfejllekofcacodljhdhibkg';
 
+/** @const {string} Adafruit's vendor id as reported by Circuit Playground boards */
+export const ADAFRUIT_VID = '0x239a';
+
+/** @const {string} The Circuit Playground product id as reported by Circuit playground boards */
+export const CIRCUIT_PLAYGROUND_PID = '0x8011';
```

Comment:
I gather ([here](http://www.voti.nl/docs/usb-pid.html), [here](https://learn.adafruit.com/hacking-the-kinect/verify-the-vid-and-pid), [here](https://learn.adafruit.com/introducing-trinket/starting-the-bootloader), [here](https://learn.adafruit.com/simple-arduino-based-usb-vid-and-pid-tester/lets-do-this-thing)) that these IDs are broadly considered stable and are the standard way to identify a USB device and look up drivers. It sounds like the Adafruit vendor id was purchased/reserved by them from the [USB developers group](http://www.usb.org/developers) and that they guarantee product id to be unique within their vendor id.

These IDs are kept [in this configuration file in this repo](https://github.com/adafruit/Adafruit_Arduino_Boards/blob/master/boards.txt) to make them available to the Arduino IDE, and while I'm not sure this is good evidence there is _very_ low churn on that file.

More generally, I think my next change should introduce some metrics around connected boards.  I want to have the maker setup page survey automatically include information about the chosen serialport in cases where we report "Board Plugged In" but not "Board Connectable" to catch conflicts like this in the future, and it'd be interesting to anonymously report port info for successfully connected boards out to GA or someplace.